tests/functional/test_wheel.py

```diff
@@ -201,6 +201,7 @@ def test_no_clean_option_blocks_cleaning_after_wheel(script, data):
         '--find-links={data.find_links}'.format(**locals()),
         'simple',
         expect_temp=True,
+        allow_stderr_warning=True,
```

Comment:
Can we add a comment saying "remove allow_stderr_warning, which is used for the --build deprecation"?

Reply:
Yep, in this case I think a comment is enough because this test will have to be revisited anyway when --build is removed.